frontend/ui/widget/dictquicklookup.lua

```diff
@@ -168,6 +171,8 @@ function DictQuickLookup:getHtmlDictionaryCss()
 
         body {
             margin: 0;
+            line-height: 1.3em;
```

Comment:
Unitless is preferable due to potential inheritance issues.

Reply:
Done, I checked it does display the same.